airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -842,6 +842,156 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  DatetimeIntervalCursor:
```

Comment:
@lazebnyi can we just add a field to the existing component's interface instead of creating an entirely new one?

Reply:
Unfortunately, we need to update the type enum in the properties, so we cannot simply use a reference to an already existing schema